src/main/java/com/dreammaster/gthandler/recipes/AlloySmelterRecipes.java

```diff
@@ -8,236 +11,343 @@
+
     @Override
     public void run() {
+        registerCoreModRecipes();
+        registerEnderIORecipes();
+        registerTinkerConstructRecipes();
+
+    }
+
+    public void registerTinkerConstructRecipes() {
+        if (!Loader.isModLoaded(modIDTinkerConstruct)) {
+            return;
+        }
+
+        ItemStack x3_AluminiumDusts = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 3L);
+
+        ItemStack x1_CopperDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Copper, 1L);
+
+        ItemStack x4_AluminiumBrassIngots = GT_ModHandler.getModItem(modIDTinkerConstruct, "materials", 4L, 14);
+
+        GT_Values.RA.addAlloySmelterRecipe(
+                x3_AluminiumDusts,
+                x1_CopperDust,
+                x4_AluminiumBrassIngots,
+                5 * SECONDS,
+                (int) TierEU.RECIPE_MV);
+
+    }
+
+    public void registerCoreModRecipes() {
+        ItemStack x4_MicaBasedSheets = CustomItemList.MicaBasedSheet.get(4L);
+
+        ItemStack x1_SiliconDioxideDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L);
+
+        ItemStack x4_MicaInsulatorSheets = CustomItemList.MicaInsulatorSheet.get(4L);
+
+        ItemStack x1_RubyDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Ruby, 1L);
+
+        ItemStack x1_SapphireDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sapphire, 1L);
+
+        ItemStack x1_GreenSapphire = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GreenSapphire, 1L);
+
+        ItemStack x1_NetherQuartzDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GreenSapphire, 1L);
+
+        ItemStack x1_CertusQuartzDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.CertusQuartz, 1L);
+
+        ItemStack x2_AluminoSilicateWools = CustomItemList.AluminoSilicateWool.get(2L);
+
+        ItemStack x2_QuartziteDusts = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Quartzite, 2L);
+
+        ItemStack x1_GlassDust = GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glass, 1);
+
+        ItemStack x1_GlassTube = ItemList.Circuit_Parts_Glass_Tube.get(1L);
+
+        ItemStack x2_IridiumReinforcedPlates = GT_ModHandler.getModItem("IC2", "itemPartIridium", 2L);
+
+        ItemStack x3_IridiumAlloyItemCasings = com.dreammaster.item.ItemList.IridiumAlloyItemCasing.getIS()
+                .splitStack(3);
+
+        ItemStack x1_Clay = new ItemStack(Items.clay_ball);
+
+        ItemStack x2_CokeOvenBricks = com.dreammaster.item.ItemList.CokeOvenBrick.getIS().splitStack(2);
+
+        ItemStack x1_ObsidianBlock = new ItemStack(Blocks.obsidian, 1, 0);
+
+        ItemStack x1_ObsidianIngot = GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Obsidian, 2L);
+
+        ItemStack shapeMoldIngot = ItemList.Shape_Mold_Ingot.get(0L);
+        ItemStack shapeMoldBall = ItemList.Shape_Mold_Ball.get(0);
+
+        GT_Values.RA.addAlloySmelterRecipe(
+                x4_MicaBasedSheets,
```

Comment:
These would be some of my ideas:
```
import static coremod.dsl.RecipeDsl.*; // put DSL utilities here
// Make these final, sorted alphabetically and implicitly x1
final ItemStack obsidianIngot = GT_OreDictUnificator.get(OrePrefixes.ingot, Materials.Obsidian, 2L);
// ...
GT_Values.RA.addXRecipe(
  items(4, obsidianIngot), // Comes from the RecipeDsl static import
  mB(144, lava), // make units clear, this would also let us add helpers like fluidIngots, fluidBuckets etc. for clearer units
  seconds(20), // 20 * SECONDS is also good, this would by a style choice
  voltAmps(TierEU.LV, 0.5f)
);
```
 - Make all the locals `final` if possible for sure
 - Add a static-importable DSL (domain-specific language) class with helper static methods and constants to keep the style consistent everywhere
 - Sort the local stacks alphabetically (just use the IDE/text editor sort lines functionality) to ensure a consistent order and clean diffs